src/backends/github/API.js

```diff
@@ -30,6 +30,15 @@ export default class API {
       });
   }
 
+  isPrivateRepo() {
+    return this.request(this.repoURL)
+      .then(repo => repo.private)
+      .catch(error => {
```

Comment:
`request()` should catch itself on an API error -- we probably don't need to catch here unless there is a good reason.

Reply:
I copied this from the `hasWriteAccess()` method above. Without knowing more about why that one is written like that, I couldn't tell you if it needs the catch or not, although yeah `request()` should probably catch it by itself. What do you think, should I keep it or remove it?

Reply:
Let's go ahead and remove it, Benaiah agreed as well.